src/mutations/ScheduledEventMutations.ts

```diff
@@ -41,9 +42,10 @@ export default class ScheduledEventMutations {
     private equipmentDataSource: EquipmentDataSource
   ) {}
 
-  @ValidateArgs(
-    createScheduledEventValidationSchema(CalendarExplicitBookableTypes)
-  )
+  @EventBus(Event.PROPOSAL_BOOKING_TIME_SLOT_ADDED)
+  // @ValidateArgs(
```

Comment:
Are the commend lines something that is left behind?

Reply:
Yes I will have to check this probably it is forgotten.